tests/spread/core22/package-repositories/task.yaml

```diff
@@ -5,15 +5,14 @@ environment:
   SNAP/test_apt_key_name: test-apt-key-name
   SNAP/test_apt_keyserver: test-apt-keyserver
   SNAP/test_apt_ppa: test-apt-ppa
+  SNAP/test_pin: test-pin
   SNAPCRAFT_BUILD_ENVIRONMENT: ""
 
 restore: |
   cd "$SNAP"
   rm -f ./*.snap
   snapcraft clean
   snapcraft clean --destructive-mode
-  ls -l /etc/apt/sources.list.d/
```

Comment:
removed this because tests/spread/tools/restore.sh handles this per-test already.